pkg/injector/sidecar/consts.go

```diff
@@ -31,9 +31,16 @@ const (
 	SidecarMetricsEnabledLabel     = "dapr.io/metrics-enabled"
 	APIVersionV1                   = "v1.0"
 	UnixDomainSocketVolume         = "dapr-unix-domain-socket" // Name of the Unix domain socket volume.
+	UserContainerAppProtocolName   = "APP_PROTOCOL"            // Name of the variable exposed to the app containing the app protocol.
```

Comment:
```suggestion
	UserContainerAppProtocolName   = "DAPR_APP_PROTOCOL"            // Name of the variable exposed to the app containing the app protocol.
```

Reply:
We use `DAPR_*` for env vars that are about the Dapr container, such as "DAPR_HTTP_PORT" -> the port Dapr's HTTP server is listening on. This instead is about the app. We already have "APP_CHANNEL_ADDRESS", "APP_ID", "APP_PORT".

Reply:
All of these are still Dapr related, and therefore must have a `DAPR` prefix. Makes sense to have the `DAPR_APP` namespace for apps using Dapr. We should deprecate these other `APP_*` env vars, in favor of `DAPR_APP_*` imo.

In my opinion, not namespacing env vars is quite hostile/bad mannered to users and generally breaks convention, especially with `APP` being so generic.

Reply:
Sounds good to me :+1: